dps/utils/pandas_utilities.py

```diff
@@ -72,6 +72,22 @@ def divide_by_series(s1, s2):
 	s = s1.div(s2)
 	return s
 
+def add_dfs(l_df):
+	'''
```

Comment:
Will this correctly add together dataframes that contain values + uncertainties?  i.e. will it just add the uncertainties together (not in quadrature) ?